src/Lesson6/CreditCard.java

```diff
@@ -0,0 +1,35 @@
+package Lesson6;
+
+public class CreditCard {
+
+    String accountNumber;
+    int balance;
+
+    public CreditCard(String accountNumber, int balance) {
+        this.accountNumber = accountNumber;
+        this.balance = balance;
+    }
+
+    public void addMoney(int balance) {
+        if (balance <= 0) {
+            return;
+        }
+        this.balance += balance;
+    }
+
+    public void withdrawMoney(int balance) {
```

Comment:
This method should return something.
If we couldn't withdraw, let's return, e.g. -1